### PR TITLE
Dev > Main - Updating version to 1.2.1

### PR DESCRIPTION
**Pull request by liamegan**
This update includes the following updates:
- Update the start function such that a server is now attached to every adaptor with an IPv4 interface; and 
- Add a page plugins system complete with page plugins package in the name of `sweet-potato-spices`.

### CODE REVIEW

**Review by marlonmarcello — APPROVED** [review]

File: cooker/package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
     "express": "^4.17.1",
     "fs-extra": "^9.1.0",
     "gray-matter": "^4.0.3",
+    "image-size": "^1.0.0",
```
Comment (marlonmarcello) [inline cooker/package.json]: We can remove this dep from the cooker

File: spices/bin/page-plugins/CWD.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const CWD = function () {
```
Comment (marlonmarcello) [inline spices/bin/page-plugins/CWD.js:1]: I am almost certain you can use `process` and `__dirname` inside pug. Need to double check.

**Review by marlonmarcello — CHANGES_REQUESTED** [review]

File: spices/bin/page-plugins/CWD.js (hunk 2 above)
Comment (liamegan) [reply spices/bin/page-plugins/CWD.js:1]: 100%. This is purely a testing plugin.

**Review by andrewrubin — APPROVED** [review]
All good here @liamegan. Are you intending to publish that CWD plugin as like an example?

File: cooker/package.json (hunk 1 above)
Comment (liamegan) [reply cooker/package.json]: This is done.

File: spices/bin/page-plugins/CWD.js (hunk 2 above)
Comment (marlonmarcello) [reply spices/bin/page-plugins/CWD.js:1]: Gotcha!

**Review by marlonmarcello — APPROVED** [review]

**Comment by liamegan** [comment]
🥳

**Pull request merged**